python/ray/scripts/scripts.py

```diff
@@ -759,90 +772,71 @@ def start(
+                    cf.bold(
+                        "  RAY_ADDRESS='http://<dashboard URL>:{}' ray job submit "
+                        "--working-dir . "
+                        "-- python my_script.py"
+                    ),
+                    ray_params.dashboard_port,
                 )
+            cli_logger.newline()
+            cli_logger.print(
+                "See https://docs.ray.io/en/latest/cluster/running-applications"
+                "/job-submission/index.html"
+            )
+            cli_logger.print(
+                "for more information on connecting to the Ray cluster from "
+                "a remote client."
+            )
+            cli_logger.newline()
+            cli_logger.print("To see the status of the cluster, use")
+            cli_logger.print("  {}".format(cf.bold("ray status")))
+            if dashboard_url:
+                cli_logger.print("To monitor and debug Ray, view the dashboard at ")
                 cli_logger.print(
-                    "dashboard {}is accessible and use the Ray Jobs API. For example:",
-                    f"({dashboard_url}) " if dashboard_url else "",
-                )
-                if dashboard_url:
-                    cli_logger.print(
-                        cf.bold(
-                            "  RAY_ADDRESS='http://<dashboard URL>:{}' ray job submit "
-                            "--working-dir . "
-                            "-- python my_script.py"
-                        ),
-                        ray_params.dashboard_port,
+                    "  {}".format(
+                        cf.bold(dashboard_url),
                     )
-                cli_logger.newline()
-                cli_logger.print(
-                    "See https://docs.ray.io/en/latest/cluster/running-applications"
-                    "/job-submission/index.html"
-                )
-                cli_logger.print(
-                    "for more information on connecting to the Ray cluster from "
-                    "a remote client."
                 )
-                cli_logger.newline()
-                cli_logger.print("To see the status of the cluster, use")
-                cli_logger.print("  {}".format(cf.bold("ray status")))
-                if dashboard_url:
-                    cli_logger.print("To monitor and debug Ray, view the dashboard at ")
-                    cli_logger.print(
-                        "  {}".format(
-                            cf.bold(dashboard_url),
-                        )
-                    )
-                cli_logger.newline()
-                cli_logger.print(
-                    cf.underlined(
-                        "If connection fails, check your "
-                        "firewall settings and "
-                        "network configuration."
-                    )
+            cli_logger.newline()
+            cli_logger.print(
+                cf.underlined(
+                    "If connection fails, check your "
```

Comment:
What does this "connection" refer to?

Reply:
Good question. @rkooo567 is it about dashboard cannot be seen?